core/server/middleware/request-id.js

```diff
@@ -0,0 +1,13 @@
+var uuid = require('uuid'),
+    _ = require('lodash'),
+    headerName = 'X-Request-Id',
+    idAttribute = 'requestId';
+
+module.exports = function requestId(req, res, next) {
+    req.userId = _.get(req, 'user.id', null);
```

Comment:
Question in the original PR was whether this line is used at all? I wasn't sure.

Totally agree we could add a customisation option to the middleware when it's moved, just not sure if it would do anything?